src/index_runner/main.py

```diff
@@ -115,6 +115,7 @@ def main():
         except ValueError as err:
             logger.error(f'JSON parsing error: {err}')
             logger.error(f'Message content: {val}')
+            continue
```

Comment:
If you just continue you'll get the same message again, won't you, since you haven't committed the offset?

Reply:
aaand you beat me to it